spec.bs

```diff
@@ -253,8 +258,8 @@ To <dfn>check interest group permissions</dfn> given an [=origin=]
 1. If |ownerOrigin| is [=same origin=] to |frameOrigin|, then return true.

```

Comment:
While you're here, would you mind adding something like:

```
1. [=Assert=]: These steps are running in parallel
```

This impacts fetch, so it would be nice to have all fetch-calling algorithms have this up front to avoid having to back-trace the callsites and see where this is coming from.